Project/ProjectForm.cs

```diff
@@ -4548,7 +4572,7 @@ public void OnWorldSelectionChanged(MapSelection sel)
                     var audiopl = sel.Audio;
                     Archetype arch = mlo?.Archetype ?? ent?.MloParent?.Archetype ?? ent?.Archetype;
                     YtypFile ytyp = mlo?.Archetype?.Ytyp ?? ent?.MloParent?.Archetype?.Ytyp ?? ent?.Archetype?.Ytyp;
-                    YmapFile ymap = ent?.Ymap ?? ent?.MloParent?.Ymap ?? cargen?.Ymap ?? grassbatch?.Ymap ?? mlo?.Ymap;
+                    YmapFile ymap = ent?.Ymap ?? cargen?.Ymap ?? grassbatch?.Ymap ?? mlo?.Ymap;
```

Comment:
ymaps should only be addable through mlo instances... that's why I removed this bit.